performance.rst

```diff
@@ -211,8 +211,8 @@ deployment process too):
 
 .. _profiling-applications:
 
-Profiling Applications
-----------------------
+Profiling Applications and Stopwatch
```

Comment:
Shouldn't this be the following?

```suggestion
Profiling Applications with Stopwatch
```

Reply:
I'm using the Stopwatch to measure how long a Console Command is taking to run. That's it. I wouldn't call that "profiling" - would you? So the message I'm trying to get across with the word "and" is: "It is a stopwatch; you can use it for profiling; but you can also just use it as a, well, stopwatch."
But I must admit that this message certainly doesn't get across fully ;-)